### PR TITLE
[rhacs-docs-3.71] Change plugin to remove hyphen per Style Guide

### PR DESCRIPTION
**Pull request by mburke5678**
Manual cherrypick of https://github.com/openshift/openshift-docs/pull/54735

- https://54863--docspreview.netlify.app/openshift-acs/latest/integration/integrate-with-email.html#configure-acs-kube-security-platform
- https://54863--docspreview.netlify.app/openshift-acs/latest/operating/manage-user-access/manage-role-based-access-control-3630.html#resource-definitions_manage-role-based-access-control
- https://54863--docspreview.netlify.app/openshift-acs/latest/operating/use-admission-controller-enforcement.html

### CODE REVIEW

**Comment by mburke5678** [comment]
@kcarmichael08 PTAL

**Comment by kcarmichael08** [comment]
Thank you so much - LGTM!

**Comment by kcarmichael08** [comment]
Actually, looking at it more closely, it looks like a merge artifact made it in - can you revert this? I'll do the manual CP if you want.

File: modules/configure-acs-kube-security-platform.adoc [hunk 1]
```diff
@@ -3,7 +3,11 @@
 // * integration/integrate-using-email.adoc
 :_module-type: PROCEDURE
 [id="configure-acs-kube-security-platform"]
+<<<<<<< HEAD
```
Comment (kcarmichael08) [inline modules/configure-acs-kube-security-platform.adoc:6]: this shows up in the preview

Comment (mburke5678) [reply modules/configure-acs-kube-security-platform.adoc]: @kcarmichael08 Sorry. It helps to save the file before committing it!

**Comment by kcarmichael08** [comment]
This looks good - thank you so much for doing this! (manual cherry picks are ugh)

**Pull request merged**